avidemux_sk.ts

```diff
@@ -10956,32 +10956,32 @@ Súbor musí byť preindexovaný. Pokračovať?</translation>
     <message>
         <location line="+460"/>
         <source>Recent Files</source>
-        <translation>Nedávne &amp;súbory</translation>
+        <translation>Nedávne súbory</translation>
```

Comment:
Do you remove the accelerator just because the source string doesn't have one? Or are there any issues with it?

(The same question applies below)

Reply:
> Do you remove the accelerator just because the source string doesn't have one?

Yes, there were no other reasons than that.

Reply:
> Yes, there were no other reasons than that.

But this removes functionality, and if there is no technical or other problem which needs to be addressed, why should this be done in the first place? To make Qt Linguist happy?

(The source has not been modified to include an accelerator due to laziness as each modification entails an update of all existing translations)

Reply:
I must admit this is totally new to me - I always thought I need to stick to what the original string uses. Will revert this change and update other string within the next few days.

Reply:
> I always thought I need to stick to what the original string uses.

There are indeed some cases where it is absolutely necessary to stick to the original, else we get incidents like [[sk] Fix crash in encoding and processing dialogs](https://github.com/mean00/avidemux2_i18n/commit/b07a52f3011b4d4aae315eb5266a93d2297969f7), just not this one.

Reply:
> > I always thought I need to stick to what the original string uses.
> 
> There are indeed some cases where it is absolutely necessary to stick to the original, else we get incidents like [[sk] Fix crash in encoding and processing dialogs](https://github.com/mean00/avidemux2_i18n/commit/b07a52f3011b4d4aae315eb5266a93d2297969f7), just not this one.

Understood. Thank you for clarification. I've just force-pushed a much larger update.